sponsorship_tracking/__manifest__.py

```diff
@@ -30,12 +29,14 @@
 # pylint: disable=C8101
 {
     'name': 'Compassion Sponsorships Tracking',
-    'version': '10.0.2.0.1',
+    'version': '11.0.0.0.0',
     'category': 'Other',
     'author': 'Compassion CH',
     'license': 'AGPL-3',
     'website': 'http://www.compassion.ch',
-    'depends': ['sponsorship_compassion'],
+    'depends': ['sponsorship_compassion',
+                'crm_compassion',
```

Comment:
Why adding `crm_compassion` as dependency?

Reply:
It must have been an error, I removed it